examples/plotting_data.py

```diff
@@ -14,24 +14,38 @@
 fig, ax = plt.subplots(figsize=(12, 5))
 ax, data = plots.eng_format(ax, data, "W", 1000)
```

Comment:
It is confusing that `eng_format` does two things at a time. Suggestion: Let it do just one thing (setting the Engineering Formatter on the axis). It would be called like this:

`ax = plots.eng_format(ax, unit)`

The conversion can happen outside.

Then you may decide to further harmonise the api of `plot_dispatch` and `plot_dispatch_plotly` by calling `eng_format` from `plot_dispatch` and asking for a unit (SI unit) as another argument, just as in `plot_dispatch_plotly`.

Reply:
Done. The only difference left between `plot_dispatch` and `plot_dispatch_plotly` is that the first additionally demands the axis as an argument.